rdr_service/genomic/genomic_job_controller.py

```diff
@@ -751,20 +751,19 @@ def reconcile_pdr_data(self):
             self.file_processed_dao,
             self.metrics_dao,
             self.manifest_file_dao,
-            self.manifest_feedback_dao,
-            self.event_dao
+            self.manifest_feedback_dao
         ]
 
         for dao in reconcile_daos:
             table_name = dao.model_type.__tablename__
             if not hasattr(dao, 'get_last_updated_records'):
                 continue
 
-            records = dao.get_last_updated_records(from_date=last_job_run)
-            if not records:
+            record_ids = dao.get_last_updated_records(from_date=last_job_run)
+            if not record_ids:
                 continue
 
-            batch_ids = [obj.id for obj in records]
+            batch_ids = [obj.id for obj in record_ids]
```

Comment:
Is `record_ids` a list of ids or objects?

Reply:
So in SQLAlchemy when using an  `.all()` even though you specified wanting only one field it still returns a list of objects with one key/value. Still have to do the list comprehension on the records, but the initial record load is significantly smaller this way.

Reply:
Ah, gotcha. Sounds good.